solr/bin/solr

```diff
@@ -1426,8 +1426,6 @@ if [ -n "${SOLR_CIRCUITBREAKER_QUERY_LOADAVG:-}" ]; then
   SOLR_OPTS+=("-Dsolr.circuitbreaker.query.loadavg=$SOLR_CIRCUITBREAKER_QUERY_LOADAVG")
 fi
 
-echo "SOLR_OPTS is now: ${SOLR_OPTS[*]}"
-
```

Comment:
Good catch about this one. Any thoughts about changes in `solr.cmd` similar to the `solr` changes at lines 1409-1427 above?

Reply:
Yea, good catch. It would be covered by #1935 but unless that is merged first, we'll definitely need it...